hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterServiceImpl.java

```diff
@@ -933,11 +954,11 @@ private void shutdownNodes() {
         hazelcastInstance.getLifecycleService().shutdown();
     }
 
-    public void initialClusterState(ClusterState clusterState) {
+    public void initialClusterState(ClusterState clusterState, Version version) {
         if (node.joined()) {
```

Comment:
Invoked from `FinalizeJoinOperation` which is sent from master to joining member (so the joining member will initialize its cluster version here). Single-node cluster is taken care of in `Node.setAsMaster` which is invoked from `Node.join` when no joiner is configured or if no other members are found (so current node self-appoints itself master).